pom.xml

```diff
@@ -125,12 +125,12 @@
 
         <!-- Quarkus -->
         <!-- Version used for Outbox extension development, changes frequently to align with latest Quarkus -->
-        <quarkus.version.extension>3.14.0</quarkus.version.extension>
+        <quarkus.version.extension>3.15.1</quarkus.version.extension>
```

Comment:
Hi, I just bumped the Outbox to 3.16.3 in #5993. I'd prefer we not change that in this PR and instead focus on the OTEL and Quarkus runtime version only to 3.15.1, to focus this commit on DS, operator, and non-extension bits.

Reply:
@Naros Updated